ts/@live-compositor/web/src/input/input.ts

```diff
@@ -0,0 +1,64 @@
+import { Frame, InputId } from '@live-compositor/browser-render';
+import MP4Source from './mp4/source';
+import { CompositorEventType } from 'live-compositor';
+import { EventSender } from '../eventSender';
+import InputSource from './source';
+import { RegisterInputRequest } from '@live-compositor/core';
+
+/**
+ * Represents frame produced by decoder. All `InputFrame`s have to be manually freed.
```

Comment:
Where?

Normally you don't want to write info where something is used, but this is potentially dangerous, so you should describe where this type is used, and where / at which point free fuction should be called